test-with-jest/.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
```

Comment:
nit: is there a reason these dotfiles are nested at the test-with-jest level? As opposed to the top level so they apply to the entire repo (this one comment applies to all dotfiles here). If they were just autogenerated by some jest initializer we can also delete them

Reply:
Moving it.